### PR TITLE
Defined "grandfathered in"

### LINKED ISSUE #320: Define 'grandfathered in'
Grandfathering comes from the [Grandfather Clause](https://en.wikipedia.org/wiki/Grandfather_clause) of the Jim Crow Era. Grandfathering is what disenfranchised Black voters.

Colloquially it is used (especially in business contexts), to mean adopted in with old rules when new rules apply elsewhere.

Helpful references:
https://www.npr.org/sections/codeswitch/2013/10/21/239081586/the-racial-history-of-the-grandfather-clause?t=1599086680635
https://www.nytimes.com/2020/08/03/us/racism-massachusetts-grandfathering.html
https://www.upworthy.com/10-common-phrases-that-are-actually-racist-af
https://medium.com/@nriley/words-matter-why-we-should-put-an-end-to-grandfathering-8b19efe08b6a

### PR DESCRIPTION
**Pull request by kelseyhuse30**
# This is the code change for issue #320 

## Index Page
image

## Definition Page
image
image

### CODE REVIEW

File: 11ty/definitions/grandfathered-in.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: grandfathered in
+slug: 'grandfathered-in'
```
Comment (tatianamac) [inline 11ty/definitions/grandfathered-in.md]:
```suggestion
title: grandfathering
slug: 'grandfathering'
```

File: 11ty/definitions/grandfathered-in.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+speech: verb
+defined: true
+flag:
+ level: avoid
+ text: racist language
+excerpt: To permit to continue under expired rules
```
Comment (tatianamac) [inline 11ty/definitions/grandfathered-in.md]:
```suggestion
excerpt: avoid. colloquially, to permit to continue under expired rules; racist origins of Grandfather Clause
```

File: 11ty/definitions/grandfathered-in.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    href: https://www.nytimes.com/2020/08/03/us/racism-massachusetts-grandfathering.html
+  - text: 10 common phrases that are actually racist AF.
+    href: https://www.upworthy.com/10-common-phrases-that-are-actually-racist-af
+  - text: "Words Matter: Why We Should Put an End to 'Grandfathering'"
+    href: https://medium.com/@nriley/words-matter-why-we-should-put-an-end-to-grandfathering-8b19efe08b6a
+  - text: The Racial History Of The 'Grandfather Clause'
+    href: https://www.npr.org/sections/codeswitch/2013/10/21/239081586/the-racial-history-of-the-grandfather-clause
+---
+
+To permit to continue under expired rules
```
Comment (tatianamac) [inline 11ty/definitions/grandfathered-in.md]:
```suggestion
colloquially, to permit to continue under expired rules
```

File: 11ty/definitions/grandfathered-in.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Issues
+
+In the United States, ‘Grandfather Clause’ originally referred to provisions adopted by seven Southern states after the Civil War in an effort to disenfranchise African-American voters by requiring voters to pass literacy tests or meet other significant qualifications, while exempting from such requirements those who were descendants of men who were eligible to vote before 1867. This type of law ensured that illiterate whites were able to vote while restricting Black Americans from voting.  This practice of grandfather clauses did not officially end until 1965, when the Voting Rights Act was implemented.
+
+## Impact
+
+The modern use of the phrase "grandfathered in" ignores its racist roots and normalizes a term that enfranchised white Americans and restricted Black Americans from voting.  Refraining to use this term is an acknowledgement of its history and a dedication to inclusivity by avoiding inciting images of racism out of context.
```
Comment (tatianamac) [inline 11ty/definitions/grandfathered-in.md]:
```suggestion
The modern use of the phrases of "grandfathering" and "grandfathered in" ignores their racist roots and normalises a term that further enfranchised white Americans and restricted Black Americans from voting.  Refraining to use this term is an acknowledgement of its history and a dedication to inclusivity by avoiding inciting images of racism out of context.
```

File: 11ty/definitions/grandfathered-in.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+## Usage Tip
+
+Use a neutral word like 'legacy' instead, or be more specific about the circumstances.  Rather than saying 'those customers were grandfathered in to the 2017 pricing schedule', rephrase it to 'those customers have retained the pricing schedule from 2017 due to the terms of service at the time' or 'those customers are on a legacy pricing plan from 2017.
```
Comment (tatianamac) [inline 11ty/definitions/grandfathered-in.md]: I might suggest here splitting this into a couple of different examples.

> When possible, be more specific about the circumstances. Usually, we can acknowledge a specific timeframe or what benefits a group is receiving during which timelines.

> Substitute "grandfathered in" for more active language. "We allowed X people Y benefits since they were long-time customers (more than 10 years).

Also, the concept of grandfathering should be considered. Why are you providing benefits and/or exceptions to this group? Are you perpetuating the inequitable concepts that this word carries as a result? Removing just the word will not remove these inequities.

Comment (kelseyhuse30) [reply 11ty/definitions/grandfathered-in.md]: Thank you! I split this out & added one additional comment regarding legality/contracts based on my own experience.  The term 'grandfathered in' was frequently used at a past employer of mine when referring to customers on older contracts which we were obligated to keep.  Happy to take it out if it doesn't add much to the definition.

I especially like your addition to reconsider why specific benefits/exceptions are allowed, and if they will continue inequities.

**Review by tatianamac — APPROVED** [review]
🚀

**Comment by tatianamac** [comment]
Thank you so much for contributing, @kelseyhuse30 ! I hope you'll come back and join us again soon!
